front/app/modules/commercial/user_custom_fields/admin/components/GenderChart.tsx

```diff
@@ -163,18 +194,30 @@ class GenderChart extends PureComponent<Props & WrappedComponentProps, State> {
             <PieChartStyleFixesDiv>
               <PieChart
                 data={serie}
+                width={164}
                 mapping={{
                   angle: 'value',
                   name: 'name',
-                  fill: ({ row }) => labelColors[row.code] ?? colorMain,
+                  opacity: ({ rowIndex }) => {
+                    if (this.state.hoverIndex === undefined) return 1;
+                    return this.state.hoverIndex === rowIndex ? 1 : 0.3;
+                  },
                 }}
                 pie={{
                   startAngle: 0,
                   endAngle: 360,
                   outerRadius: 60,
                 }}
                 innerRef={this.currentChart}
-                annotations
```

Comment:
I see you removed the `annotations` prop. I guess this one was not working, right? It's still being used in `app/containers/Admin/dashboard/users/charts/PieChartByCategory.tsx` though, so I assume it's broken there too. I suggest we remove it there as well and add the same tooltip behavior, and the remove the `annotations` prop from the `PieChart` component altogether. WDYT?

Reply:
It wasn't working at first, it got fixed as i work on adding the legend but i removed it to make it look as the other pie charts.
Didn't knew we have that other pie chart! yes i would have to add it here and remove the annotation prop.